tests/cpp/data/test_sparse_page_dmatrix.cc

```diff
@@ -59,34 +59,25 @@ TEST(SparsePageDMatrix, ColAccess) {
   xgboost::DMatrix * dmat = xgboost::DMatrix::Load(
     tmp_file + "#" + tmp_file + ".cache", true, false);
   std::remove(tmp_file.c_str());
-  EXPECT_FALSE(FileExists(tmp_file + ".cache.col.page"));
 
```

Comment:
We may want to test the existence of `col.page` file. Or does `col.page` file not generated at all?

Reply:
Done.